report/renderer/details-renderer.js

```diff
@@ -53,7 +53,12 @@ export class DetailsRenderer {
         return this._renderList(details);
       case 'table':
       case 'opportunity':
-        return this._renderTable(details);
+      {
+        // Defer rendering of hidden tables, for a faster FCP
```

Comment:
This is a really good idea in general, but feels like there could be a more principled way of doing it. What about deferring all audit detail rendering to a second pass and triggering it up in `renderReport()`?

Reply:
Good idea. We probably need a special case for filmstrip (and maybe *-budgets) but.. otherwise all of detailsrenderer deferred seems cool.